MqttClient_Hello/app/application.cpp

```diff
@@ -38,6 +38,7 @@ void onMessageReceived(String topic, String message)
 // Run MQTT client
 void startMqttClient()
 {
+	mqtt.setWill("last/will","The connection from this device is lost:(", 1, true);
```

Comment:
You are correct, I thought I saw an unclosed open bracket but it was a sad face instead!